db/migrate/20210810114405_create_host_initiator_groups.rb

```diff
@@ -0,0 +1,16 @@
+class CreateHostInitiatorGroups < ActiveRecord::Migration[6.0]
+  def change
+    create_table :host_initiator_groups do |t|
+      t.references :ems, :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :physical_storage, :type => :bigint, :index => true
+      t.references :host_initiator, :type => :bigint, :index => true
```

Comment:
I was assuming a group would has_many initiators, but unless I'm reading it wrong, this makes it a 1:1

Reply:
yeah I think you want to add `t.references :host_initiator_group` to `:host_initiators` assuming you want a group to have >1 initiator

Reply:
Sorry, this has been an oversight.